content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,266 @@
+
+Technology Partners can add up to eight images (seven if you are including a video) in an integration tile's media carousel. When naming image files, do not use spaces. Instead, use underscores (`_`). 
+
+The images must meet the following requirements:
+
+| Image Requirements | Description                                                                                                                                       |
+|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Type               | `.jpg` or `.png`.                                                                                                                                 |
+| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
+| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
+| Name               | Use letters, numbers, underscores, and hyphens. Spaces are not allowed.                                                                           |
+| Color Mode         | RGB                                                                                                                                               |
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the media carousel which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][18].
+
+### Open a pull request
+
+Open a pull request that contains your integration tile's asset files (including images) in the [`marketplace` repository][15]. Automatic tests run checks in Azure DevOps pipelines to verify that your pull request is in good shape and contains all the required content to be updated.
+
+To request access to the Azure DevOps pipeline, leave a comment in the pull request requesting access.
+
+### Obtain approval to merge
```

Comment:
```suggestion
### Review Process
```
This originally sounded like the partner has the ability to merge once they get approval, which is not the case